### ISSUE TITLE
unable to reproduce accuracy of bvlc-alexnet

### ISSUE THREAD

**Issue by dongxiao92** [BVLC/caffe#6065]
### Issue summary
I use model definition(models/bvlc_alexnet/train_val.prototxt) and training protocol(models/bvlc_alexnet/solver.prototxt) provided by caffe to train alexnet on ImageNet.The only modification  I made is replacing mean.binaryproto with mean values computed by Facebook.And I have checked they are very close: 103.894,116.555,122.579 vs 103.94,116.78,123.68 .
But the accuracy on validation set I got is 55.06% which is observably low compared to the one reported by caffe comunity(57.1%,https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet).
The differences of mean values are tiny so I think there are other reasons making such a large drop in accuracy.
Then I use pre-trained caffemodel provided by caffe(http://dl.caffe.berkeleyvision.org/bvlc_alexnet.caffemodel) and get the accuracy.It still fall behind the reference accuracy, 55.9% vs 57.1%.
The imagenet lmdb dataset are created by script at  examples/imagenet/create_imagenet.sh with RESIZE=true.
Could anyone give some suggestions about this problem?
### Steps to reproduce
//train alexnet using caffe
./build/tools/caffe train -gpu 0 -solver models/bvlc_alexnet/solver.prototxt
//use pre-trained alexnet model
./build/tools/caffe test -gpu 0  -model models/bvlc_alexnet/train_val.prototxt -weights models/bvlc_alexnet.caffemodel
### Your system configuration
Operating system:Ubuntu 16.04
Compiler:nvcc-8,gcc-5.4.0
CUDA version (if applicable):CUDA8
CUDNN version (if applicable):cudnn515
BLAS:atlas ,cublas8.0

**Comment by Noiredd** [comment]
Please do not post usage, installation, or modeling questions, or other requests for help to Issues. Use the [caffe-users list][1] instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe. Please read the [guidelines for contributing][2] before submitting an issue or a pull request.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
